js-challenges/challenge4/answer.js

```diff
@@ -15,38 +18,40 @@ function longestWord(word){
   a.forEach(function(c){
     switch (true) { // you should probably switch on an actual expression rather than a boolean
       case c.length > b.length:
-        b = c // you should do b = c
+        b = c
         longWords.push(c)
         if (c.length > 1){
-          description = " is the longest word and is " + c.length + " characters"
+          description = singularDesc + c.length + " characters"
```

Comment:
have a go at new ES6 syntax for this if you can. google it. but it's basically syntax that looks like this
```
let word = 'hello'
`${word}` // outputs hello
```

Reply:
sorry, whats the advantage? dont get the difference between just saying word?